### PR TITLE
Added Readme Files

### PR DESCRIPTION
**Pull request by Buddhad**
- I have added the `Readme Files`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,110 @@
...
+[![Welcome to my profile](https://img.shields.io/badge/Hello,Programmer!-Welcome-blue.svg?style=flat&logo=github)](https://github.com/Ggita/Bhagavad-Gita-App)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/gita/Bhagavad-Gita-App)
+![License](https://img.shields.io/badge/License-MIT-red)
+![Stars](https://img.shields.io/github/stars/gita/Bhagavad-Gita-App?style=flat&logo=github)
+![Forks](https://img.shields.io/github/forks/gita/Bhagavad-Gita-App?style=flat&logo=github)
+
+</div>
+
+<div align="center">
+  <img height=240 src="../Bhagavad-Gita-App/bhagavad_gita/assets/images/gita.png" alt="Bhagavad-Gita-Image">
```
Comment (Gupta-Anubhav12) [inline README.md:17]: This is broken somehow

File: README.md [hunk 2]
```diff
@@ -1,2 +1,110 @@
...
+</div>
+
+<br>
+
+<div align="center">
+<img src ="https://forthebadge.com/images/badges/winter-is-coming.svg"/>
+  <img src="https://forthebadge.com/images/badges/for-you.svg" />
+
+ <img src="https://forthebadge.com/images/badges/built-for-android.svg" alt="android">
+  <img src="../Bhagavad-Gita-App\bhagavad_gita\assets\images\made-with-dart.svg" alt="js">
```
Comment (Gupta-Anubhav12) [inline README.md:27]: This too is not being rendered, bad urls probably

**Review by Gupta-Anubhav12 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Buddhad) [reply README.md:27]: Ok I will change that

Comment (Gupta-Anubhav12) [reply README.md:27]: Many Thanks

Comment (Buddhad) [reply README.md:27]: I have raised a new `PR` for this, please check it out. @Gupta-Anubhav12

**Review by ravinder-chadha — COMMENTED** [review]
You may close this as we have new updated PR for the same

**Pull request closed without merging**